src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveEvaluator.cs

```diff
@@ -74,18 +75,20 @@ public sealed class RInteractiveEvaluator : IInteractiveEvaluator {
 
         public async Task<ExecutionResult> ResetAsync(bool initialize = true) {
             try {
-                CurrentWindow.TextView.VisualElement.SizeChanged -= VisualElement_SizeChanged;
+                if (CurrentWindow != null) {
```

Comment:
No. For methods that are called from InteractiveWindow it is always `true`, but for methods that are called from RHost it is possible that CurrentWindow is null already.

---
In reply to: [64487770](https://github.com/Microsoft/RTVS/pull/1812#discussion_r64487770) [](ancestors = 64487770)